core-bundle/src/Filesystem/FilesystemItemIterator.php

```diff
@@ -17,6 +17,17 @@
  */
 class FilesystemItemIterator implements \IteratorAggregate
 {
+    public const SORT_BY_PATH_ASC = 'name_asc';
+    public const SORT_BY_PATH_DESC = 'name_desc';
```

Comment:
Would this use natural sorting (so that `img1 < img2 < img10`)?
If not should we add seperate sorting modes for natural sorting?

Reply:
It currently uses `strnatcasecmp`. Adding more sort modes would be quite easy and probably helpful. 

Any naming suggestions? :see_no_evil:

Reply:
This would then match https://www.php.net/manual/en/array.constants.php

Reply:
implemented in bdf3056b22ddbed0d073460602a271e757170a92